extensions/visualizations/oppia-visualization-sorted-tiles.directive.spec.ts

```diff
@@ -19,37 +19,97 @@
 
 require('visualizations/oppia-visualization-sorted-tiles.directive.ts');
 
+import { UpgradedServices } from 'services/UpgradedServices';
+
+import { IAnswerStatsBackendDict, AnswerStatsObjectFactory } from
+  'domain/exploration/AnswerStatsObjectFactory';
+
 describe('Oppia sorted tiles visualization', function() {
-  let $compile, $rootScope, element: JQuery;
+  let $compile, $rootScope;
+  let answerStatsObjectFactory: AnswerStatsObjectFactory;
 
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.inject(function(_$compile_, _$rootScope_) {
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
```

Comment:
Would be nice if you can remove the usage of ugs. /cc @srijanreddy98.

Reply:
Done.